src/App.js

```diff
@@ -1,7 +1,10 @@
+import React from 'react';
+import './App.css';
+
 function App() {
   return (
     <div className="App">
-      <h1>Hello</h1>
+      <h1 className='title'>Hello</h1>
```

Comment:
- [ ] I got an eslint error here that prevents the app from compiling:

![Screenshot from 2021-01-16 00-46-46](https://user-images.githubusercontent.com/4246637/104786183-ec875300-5794-11eb-8e93-79b37f5d6356.png)
Kindly fix the linter issue.

Reply:
now I realize that all the issue is because i don't use double quotes